### PR TITLE
Add Evmos testnet network

### PR DESCRIPTION
**Pull request by datradito**
Endpoints for verification:

evmos_9000-2(Olympus Mons Incentivized Testnet / Evmos Testnet): https://evmos-evm-rpc.tk/

Documentation:
[https://evmos.dev/testnet/join.html](https://evmos.dev/testnet/join.html)

### CODE REVIEW

File: src/assets/v1.3.0/compatibility_fallback_handler.json [hunk 1]
```diff
@@ -31,7 +31,8 @@
     "80001": "0xf48f2B2d2a534e402487b3ee7C18c33Aec0Fe5e4",
     "333999": "0xf48f2B2d2a534e402487b3ee7C18c33Aec0Fe5e4",
     "1313161554": "0xf48f2B2d2a534e402487b3ee7C18c33Aec0Fe5e4",
-    "1313161555": "0xf48f2B2d2a534e402487b3ee7C18c33Aec0Fe5e4"
+    "1313161555": "0xf48f2B2d2a534e402487b3ee7C18c33Aec0Fe5e4",
+    "evmos_9000-2": "0xf48f2B2d2a534e402487b3ee7C18c33Aec0Fe5e4"
```
Comment (rmeissner) [inline src/assets/v1.3.0/compatibility_fallback_handler.json:35]: Shouldn't this be just `9000`?

Comment (datradito) [reply src/assets/v1.3.0/compatibility_fallback_handler.json]: Yes I'm sorry, I was using the Network Chain ID. It's solved now.

Comment (rmeissner) [reply src/assets/v1.3.0/compatibility_fallback_handler.json]: No, we will verify the deployment later this week.

Comment (datradito) [reply src/assets/v1.3.0/compatibility_fallback_handler.json]: Hi @rmeissner , I'll have to close this PR due a critical flaw on the testnet. Thank you.
image

**Pull request closed without merging**